Update wave 4 python packages

nickzhums opened the pull request.

jsntcy commented:
/check-enforcer override

jsntcy commented:
/check-enforcer evaluate

nickzhums commented:
@sima-zhu CI builds are failing due to broken links and seem to be unrelated to the changes here - would you please take a look when you are back in office? thanks!

nickzhums commented:
/check-enforcer evaluate

weshaggard commented:
/azp run azure-sdk - ci

weshaggard commented on _data/releases/latest/python-packages.csv:
```diff
@@ -133,7 +133,7 @@
 "azure-media-nspkg","1.0.0","","azure-media-nspkg","","NA","NA","NA","","false","",""
 "azure-mgmt-azureadb2c","","1.0.0b1","azure-mgmt-azureadb2c","","NA","NA","NA","","false","",""
-"azure-mgmt-portal","","1.0.0b1","azure-mgmt-portal","","NA","NA","NA","","false","",""
+"azure-mgmt-portal","","1.0.0b1","azure-mgmt-portal","","NA","NA","NA","","true","",""
```
While this might get the packages to show up in the "mgmt" tab it will not include source links. You need to eat least set the RepoPath field in order to get the source links working. The MSDocs and GHDocs links will fix themselves the next time the automation runs assuming those doc links exist.

sima-zhu commented:
@scbedd Are these broken links affected by preview readme refactor? I can have another PR to fix the links if this is expected to you.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=672143&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=7dae026f-9e99-5d65-075e-3f7579577f94&l=1800

weshaggard commented:
@sima-zhu I fixed the broken links in https://github.com/Azure/azure-sdk/pull/2186 as the automation marked them as NA. So the repo is working but I do think it is worth following up about the pre docs issues that caused them to break to make sure that is expected.

@nickzhums as part of making the other tweaks you will need to also resolve the file conflict.

nickzhums commented:
> @sima-zhu I fixed the broken links in #2186 as the automation marked them as NA. So the repo is working but I do think it is worth following up about the pre docs issues that caused them to break to make sure that is expected.
> 
> @nickzhums as part of making the other tweaks you will need to also resolve the file conflict.

thanks, just fixed the conflicts as well as adding repo path

weshaggard commented on _data/releases/latest/python-packages.csv:
```diff
@@ -132,8 +132,8 @@
 "azure-media-nspkg","1.0.0","","azure-media-nspkg","","NA","NA","NA","","false","",""
-"azure-mgmt-azureadb2c","","1.0.0b1","azure-mgmt-azureadb2c","","NA","NA","NA","","false","",""
-"azure-mgmt-portal","","1.0.0b1","azure-mgmt-portal","","NA","NA","NA","","true","",""
+"azure-mgmt-azureadb2c","","1.0.0b1","azure-mgmt-azureadb2c","","azureadb2c","NA","NA","","false","",""
```
Did you want to set New =true for this entry.

nickzhums replied:
yes

weshaggard commented on _data/releases/latest/python-packages.csv:
```diff
@@ -200,15 +200,15 @@
 "azure-mgmt-azurestack","0.1.0","","Resource Management - Azure Stack","Stack","azurestack","NA","","mgmt","true","",""
 "azure-mgmt-azurestackhci","1.0.0","1.0.0rc1","Resource Management - Azure Stack HCI","Stack HCI","NA","","","mgmt","true","",""
-"azure-mgmt-avs","1.0.0","","Resource Management - Azure VMware Solution","VMware Solution","NA","","","mgmt","false","",""
+"azure-mgmt-avs","1.0.0","","Resource Management - Azure VMware Solution","VMware Solution","NA","","","mgmt","true","",""
```
There is no repopath for this one?

nickzhums replied:
same here, i can't find it in the repo

nickzhums replied:
@00Kai0  any idea why it isn't in repo?

weshaggard commented on _data/releases/latest/python-packages.csv:
```diff
@@ -200,82 +200,82 @@
 "azure-mgmt-batchai","2.0.0","","Resource Management - Batch AI","Batch AI","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-batchai_2.0.0/azure-mgmt-batchai/","https://docs.microsoft.com/python/api/overview/azure/batchai","NA","mgmt","false","",""
-"azure-mgmt-billing","1.0.0","","Resource Management - Billing","Cost Management + Billing","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-billing_0.2.0/azure-mgmt-billing/","","","mgmt","false","",""
-"azure-mgmt-botservice","0.2.0","","Resource Management - Bot Service","Bot Service","botservice","NA","","mgmt","false","",""
+"azure-mgmt-billing","1.0.0","","Resource Management - Billing","Cost Management + Billing","https://github.com/Azure/azure-sdk-for-python/tree/azure-mgmt-billing_0.2.0/azure-mgmt-billing/","","","mgmt","true","",""
```
We should update this to have a RepoPath from master. assuming it is in master.

nickzhums replied:
changed repopath to just "billing"

weshaggard commented:
A number of changes in this PR broken some automation so I either fixed or disabled them in PR https://github.com/Azure/azure-sdk/pull/2223 please have a review over that PR.
